reporting/CassandraBackend.cpp

```diff
@@ -652,8 +652,9 @@ void
 writeBook2(WriteBookCallbackData& cb)
```

Comment:
While you're in here, can we just change all of these things that are named "books2" to just "books"? And remove the old books stuff.

Reply:
Same here